src/framework/components/element/component.js

```diff
@@ -1001,6 +1012,14 @@ class ElementComponent extends Component {
         this.entity.setLocalPosition = Entity.prototype.setLocalPosition;
     }
 
+    /**
+     * Patched method for setting the position.
+     *
+     * @param {number|Vec3} x - The x coordinate or Vec3
+     * @param {number} y - The y coordinate
+     * @param {number} z - The z coordinate
+     * @returns {void}
```

Comment:
This line is redundant so best to remove it.

Reply:
It triggered a linting error because of `return Entity.prototype.setPosition.call(this, x, y, z);`, so I just changed that now to make it clearer that there is really no return value.